src/pages/Login/Component/MainForm/MainForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import Input from '../../../../Component/Input/Input';
+import InputWrap from '../../../../Component/InputWrap/InputWrap';
+import Button from '../../../../Component/Button/Button';
+import IconButton from '../../../../Component/IconButton/IconButton';
+import './MainForm.scss';
+
+const MainForm = props => {
+  const { onChange } = props;
+  const navigate = useNavigate();
+  const [userInfo, setUserInfo] = useState({
+    userId: '',
+    userPw: '',
+  });
+
+  const { userId, userPw } = userInfo;
+  const onChangeCheck = e => {
+    const { name, value } = e.target;
+
+    setUserInfo({ ...userInfo, [name]: value });
+  };
+
+  const onSubmit = e => {
+    e.preventDefault();
+  };
+
+  const loginValidation = e => {
+    const idRegExp = /^[A-Za-z0-9]{6,}$/;
+    const passwordRegEx =
+      /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,20}$/;
+
+    if (!idRegExp.test(userId)) {
+      // alert('6글자 이상 소문자와 숫자로 혼합된 아이디를 입력해주세요.');
+      return false;
+    } else if (!passwordRegEx.test(userPw)) {
+      // alert(
+      //   '비밀번호는 영문, 숫자, 특수기호를 조합하여 8~20글자 이상 입력해주세요.',
+      // );
+      return false;
+    } else {
+      return true;
+    }
+  };
+
+  return (
+    <div className="loginContainer">
+      <div className="loginSubTitle">
+        <h2>
+          TeaTime에 오신 것을 환영합니다. <br />
+          로그인 해주세요.
+        </h2>
+      </div>
+      <form className="loginForm" onChange={onChangeCheck} onSubmit={onSubmit}>
+        <fieldset>
+          <legend className="legend">로그인</legend>
+          <InputWrap>
+            <Input
+              placeholder="아이디 입력"
+              type="text"
+              scale="large"
+              name="userId"
+            />
+            <Input
+              placeholder="비밀번호 입력 (영문, 숫자, 특수문자 조합)"
+              type="password"
+              scale="large"
+              name="userPw"
+            />
+          </InputWrap>
+          <Input
+            className="checkbox"
+            type="checkbox"
+            text="아이디 저장"
+            onChange={onChange}
+          />
+          <Button
+            className="loginBtn"
+            scale="large"
+            shape="fill"
+            color="primary"
+            fullWidth="true"
+            disabled={!loginValidation() ? true : false}
```

Comment:
- 위 리뷰 내용 참고
- falsy일 때 true, 반대일 때 false라면 조건의 반댓값으로 넣어주면 됨

```suggestion
            disabled={!isLoginInputValid}
```